src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,12 @@
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
+    public static User findByEmail(String email) {
+        for (int i = 0; i < users.length; i++) {
+            if (email == users[i].getEmail()) {
```

Comment:
is it correct to compare strings like this?

Reply:
No. I fixed all and pulled it again. Link in file changed.